adoc/deployment-vmware.adoc

```diff
@@ -306,55 +289,18 @@ terraform plan
 terraform apply
 ----
 
-.Skip self-signed certificate verification
-[TIP]
-In case you are using a self-signed certificate you need to skip its verification, as Terraform vsphere provider is not supporting it, thus `terraform plan` will fail. To do this, type the following in your terminal session: `export VSPHERE_ALLOW_UNVERIFIED_SSL=true`
-
 ==== Setup by hand
-
-. Right-click on the template and select menu:New VM from This Template...[] and name it something like `caasp4-master-0`.
-.. Select storage `datastore` and the designated resource pool.
-
-Repeat these steps until you have created the desired number of machines comprising your cluster.
+For each VM deployment follow {ay} installation method used for deployment on Bare-metal machines as described at <<deployment.on.bare.metal>>.
```

Comment:
No idea if used `<<deployment.on.bare.metal>>` reflink will work.

Reply:
doesn't work - CI is failing due ^

Reply:
Fixed the link. Many thanks for updating the content, this is a big help! :heart: